substrate/frame/system/src/lib.rs

```diff
@@ -1032,6 +1030,12 @@ pub mod pallet {
 					})
 				}
 			}
+			#[cfg(feature = "experimental")]
+			if let Call::do_task { ref task } = call {
+				if task.is_valid() {
```

Comment:
Can you maybe mention in the doc of this function that it **MUST** be cheap, or there is a Dos vector IIUC?  
Like "no storage reads" cheap would be best, but probably impossible...

Reply:
You mean that `is_valid` check should be cheap, correct?

Reply:
Yes.

Reply:
Done